app/build.gradle

```diff
@@ -14,6 +14,7 @@ android {
         versionName "1.0"
 
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
```

Comment:
It's really weird 🤓  We already use `androidx.test.runner.AndroidJUnitRunner`, so why do we need to add the  `android.support.test.runner.AndroidJUnitRunner` 🤔

Reply:
@hoangnguyen92dn The redundant one, let me remove it 🙏